virtcontainers/container.go

```diff
@@ -972,6 +972,7 @@ func (c *Container) update(resources specs.LinuxResources) error {
 
 	newResources := ContainerResources{
 		VCPUs: uint32(utils.ConstraintsToVCPUs(*resources.CPU.Quota, *resources.CPU.Period)),
+		Mem:   uint32(*resources.Memory.Limit >> 20),
```

Comment:
please add a comment explaining this

Reply:
Please tell me what do I need to explain. Why we need Mem field or why we have 20 right shift?  ^^

Reply:
It is a little unclear as `resources.Memory.Limit` is an `int64` byte value. It would help if `ContainerResources.Mem` was called `ContainerResources.MemMb` imho.

Reply:
Yes, you are rignt. I will change the name.

Reply:
@clarecch 
> why we have 20 right shift?

Reply:
/cc @miaoyq regarding the int64 / int32 `Mem` size issue.